css-three-column-layout/styles.css

```diff
@@ -0,0 +1,64 @@
+h3,
+h2 {
+  font-family: Lato, sans-serif;
+  color: white;
+}
+
+h3 {
+  font-size: 20px;
+  line-height: 0em;
+  padding-top: 16px;
+}
+
+p {
+  font-family: Lato, sans-serif;
+  font-size: 14px;
+  line-height: 1.5em;
+  color: #d8d9da;
+  padding: 0 29px;
+}
+
+body {
+  background-color: #219bdd;
+}
+
+.container {
+  width: 900px;
+  margin: 50px auto 0;
+}
+
+.row {
+  display: flex;
+  justify-content: space-between;
+}
+
+.column-full {
+  width: 100%;
+  text-align: center;
+}
```

Comment:
Column classes should only include `flex-basis` or `width`, anything else should be a separate class. You need to keep your layout classes untainted and reusable.